valohai/metadata.py

```diff
@@ -1,9 +1,14 @@
 import json
+import os
 from typing import Any, Dict
 
 _supported_types = [int, float]
 
 
+def is_valohai_deployment():
+    return os.environ.get('VALOHAI_PORT') or \
+        os.path.exists("/valohai/valohai-metadata.json")
```

Comment:
Please move this to `valohai.config`, to sit next to `is_running_in_valohai()` :)

Also, prefer parens instead of backslashes:

```suggestion
def is_valohai_deployment() -> bool:
    return (
        os.environ.get('VALOHAI_PORT') or
        os.path.exists("/valohai/valohai-metadata.json")
    )
```